### ISSUE TITLE
Ghostty replaces \t with 4 spaces

### ISSUE THREAD

**Issue by daniilr** [ghostty-org/ghostty#3394]
I am running Ghostty 1.0 on a mac. My shell is zsh. I was able to reproduce the issue in bash as well. The default Terminal.app works good. 

https://github.com/user-attachments/assets/cc5deea1-fce1-4bad-a800-e4c92f339d81
`➜  ~ echo -e 'test\u9test'
test    test
`

Is this behaivour expected?

**Comment by qwerasd205** [comment]
Yes this is expected behavior. It's not actually 4 spaces, it's simply aligning to the next tabstop. Perhaps it would be a good quality of life feature to encode the tabs as state in a way that lets the user copy them as tabs instead of spaces though.

**Comment by mitchellh** [comment]
> Perhaps it would be a good quality of life feature to encode the tabs as state in a way that lets the user copy them as tabs instead of spaces though.

Interesting.

Converting to a feature request discussion and renaming. I'd be curious next what other terminals do here. I'm willing to do something different, just curious!
